build

```diff
@@ -61,9 +61,22 @@ log() {
     echo "===< $msg"
 }
 
+delete_unwanted_files() {
+    if [ -e "${1}" ]; then
+        log "Deleting file: ${1}"
+        rm -f "${1}"
+    else
+        log "Cannot delete file: ${1} -- file not found"
+    fi
+}
+
 make_rel() {
     # shellcheck disable=SC1010
-    ./rebar3 as "$PROFILE" do release,tar
+    ./rebar3 as "$PROFILE" do release
+    # delete outdated cert store
+    delete_unwanted_files _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
+    # shellcheck disable=SC1010
+    ./rebar3 as "$PROFILE" do tar
```

Comment:
```suggestion
    ./rebar3 as "$PROFILE" release
    # delete outdated cert store
    delete_unwanted_files _build/"${PROFILE}"/rel/emqx/lib/certifi*/priv/cacerts.pem
    ./rebar3 as "$PROFILE" tar
```

Reply:
Same as below, SC complains about both places where SC1010 is not suppressed